crates/bevy_render/src/render_resource/shader.rs

```diff
@@ -640,1312 +378,3 @@ impl From<&'static str> for ShaderRef {
         Self::Path(AssetPath::from(path))
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
what happened to all the tests?

Reply:
they've been stolen and moved into naga_oil along with the functionality that was being tested